--- a/.vite/build/workspace-root-drop-handler-B7KjFQ8M.js
+++ b/.vite/build/workspace-root-drop-handler-B7KjFQ8M.js
@@ -35828,6 +35828,7 @@
     i = e.y(n.version);
   return (
     e.r({
+      enabled: !1,
       dsn: e.v,
       environment: ME.buildFlavor,
       release: i,
--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -57032,5 +57032,6 @@
     r = e.buildFlavor !== `prod`;
   (XA({
+    enabled: !1,
     dsn: iB,
     environment: e.buildFlavor,
     release: aB(n.version),
